src/main/java/com/hobbycollection/service/CollectionService.java

```diff
@@ -0,0 +1,26 @@
+package com.hobbycollection.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class CollectionService implements ICollectionService{
+    @Override
+    public Collection getItemByID(int id) {
+        return null;
+    }
+
+    @Override
+    public Collection save(Collection collection) throws Exception {
+        return collection;
+    }
+
+    @Override
+    public ArrayList<Collection> fetchAll() {
```

Comment:
I added Service Class that implements the service Interface.
Sources:
     * Lecture (Videos)
     * https://martinfowler.com/articles/microservices.html

Reply:
This is incomplete functionality... and only features which meet the Definition of Done should be merged to master.  So, I would not merge this in its current state.